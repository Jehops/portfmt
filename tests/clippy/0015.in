OPTIONS_DEFINE=		PGSQL_SERVER A
A_VARS_OFF=		WANT_PGSQL=""
PGSQL_SERVER_USE=	GNOME=vte3
PGSQL_SERVER_VARS=	WANT_PGSQL+="server contrib" _IGNORED=bar WHATSTHIS=""

.include <bsd.port.mk>
