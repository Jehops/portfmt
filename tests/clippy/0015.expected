# Options definitions
OPTIONS_DEFINE

# Options helpers
A_VARS_OFF
PGSQL_SERVER_USE
PGSQL_SERVER_VARS

# Unknown variables
# WARNING:
# Portclippy did not recognize the following variables.
# They could be local variables only, misspellings of
# framework variables, or Portclippy needs to be made aware
# of them.  Please double check them.
#
# Prefix them with an _ to tell Portclippy to ignore them.
# This is also an important signal for other contributors
# who are working on your port.  It removes any doubt of
# whether they are framework variables or not and whether
# they are safe to remove/rename or not.

# Unknown variables in options helpers
USE_GNOME             in PGSQL_SERVER_USE
                      missing USES=gnome ?
WANT_PGSQL            in A_VARS_OFF
                      in PGSQL_SERVER_VARS
                      missing USES=pgsql ?
WHATSTHIS             in PGSQL_SERVER_VARS
